### PR TITLE
Rc1.1.1

### PR DESCRIPTION
**Pull request by coderidge**
changes to READ.me

### CODE REVIEW

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Listener/RequestListener.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function onKernelRequest(GetResponseEvent $event)
         if ('/api/doc' === $event->getRequest()->getRequestUri()) {
             return;
         }
-
```
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Listener/RequestListener.php]: Don't think you meant to commit this.

Comment (coderidge) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Listener/RequestListener.php]: yes I added that to git ignore still seemed to add to git.

Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Listener/RequestListener.php]: It's part of the source tree, so you have to explicitly remove it

Use `git add -i` in future to see what you are adding

**Comment by brettminnie** [comment]
You need to either revert the commit on the listener, or fix the changes and then re-commit it, can't merge with the build being broken

**Comment by brettminnie** [comment]
:+1:

File: .gitignore [hunk 2]
```diff
@@ -12,3 +12,4 @@
 /bin/
```
Comment (brettminnie) [inline .gitignore]: these meant to be an absolute path? `/bin/` will look off the route of the filesystem, 'bin/' will look off the root based on where the .gitignore file is placed

File: README.md [hunk 3]
```diff
@@ -137,26 +137,9 @@ Documentation can be found at http://<your installation url>/api/docs
 
 ### Daily Reports 
 
-Daily reports are in .CSV format collated from survey data for the day (editing transfer times etc, require public key of your computer to be registered on the rackspace server, username setup and firewall access granted for your ipaddress).  
+Daily reports are in .CSV format collated from survey data for the day (editing transfer times etc, require public key of your computer to be registered on the server, username setup and firewall access granted for your ipaddress).
```
Comment (brettminnie) [inline README.md]: `require public key of your computer`

Should it not rather read

`requires a public key from your computer`

**Comment by coderidge** [comment]
Edited Readme.md file to include details of daily reports

**Comment by coderidge** [comment]
typo mistakes corrected

**Pull request merged**